### DISCUSSION TITLE
Changing cooordinator IP Address

### DISCUSSION THREAD

**Discussion by yarafie** [Koenkk/zigbee2mqtt#23179] [General]
I read the manual and the faqs about when I need to repair devices but did not find the answer.

I'm planning to change my coordinator IP Address from its current 192.168.100.x to 10.10.50.x

I will update the config file of course but will I need to repair all devices again?

**Comment by duncanhancox** [comment]
no you wont need to repair/re-add devices for a change of ip address or a firmware update
